inc/customizer.php

```diff
@@ -94,7 +94,19 @@ function primer_customize_register( $wp_customize ) {
 		'priority' => 1,
 	) );
 
-	// Add page background color setting and control.
+	// Add menu background color setting and control.
+	$wp_customize->add_setting( 'menu_background_color', array(
+		'default'           => $color_scheme[3],
```

Comment:
@chriswallace This should be `$color_scheme[2]` according to https://github.com/godaddy/wp-primer-theme/pull/19/files#diff-b8746809c04b1698add2a397c385bc9eR370